### PR TITLE
chore(in-app-messaging): bump versions

### PR DESCRIPTION
**Pull request by cshfang**
#### Description of changes

Bumps notifications and dependency versions. The previous merge missed bumping dependency versions and the tag got out of sync.

#### Checklist

- [x] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-js/blob/main/CONTRIBUTING.md#steps-towards-contributions)
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/notifications/package.json [hunk 1]
```diff
@@ -42,8 +42,8 @@
 	},
 	"homepage": "https://docs.amplify.aws/",
 	"dependencies": {
-		"@aws-amplify/cache": "4.0.31",
-		"@aws-amplify/core": "4.3.11",
+		"@aws-amplify/cache": "4.0.33",
+		"@aws-amplify/core": "4.3.13",
```
Comment (elorzafe) [inline packages/notifications/package.json:46]:
```suggestion
		"@aws-amplify/cache": "4.0.35",
		"@aws-amplify/core": "4.4.0",
```

File: packages/notifications/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@aws-amplify/notifications",
-	"version": "0.2.0",
+	"version": "0.2.1",
```
Comment (elorzafe) [inline packages/notifications/package.json:3]: Checking on npm I saw latest been on `0.1.1` maybe I am missing something on our release process

File: packages/notifications/package.json (hunk 1 above)
Comment (elorzafe) [reply packages/notifications/package.json:46]: This should be same version so lerna will make the bump automatically

**Review by elorzafe — CHANGES_REQUESTED** [review]
Need to update the version of the dependencies so lerna can bump those automatically

File: packages/notifications/package.json (hunk 1 above)
Comment (cshfang) [reply packages/notifications/package.json:46]: On this branch, `4.0.33` and `4.3.13` are the updated versions

Comment (elorzafe) [reply packages/notifications/package.json:46]: Good catch! makes sense cool

**Review by elorzafe — APPROVED** [review]
LGTM! thanks @cshfang 🌮

**Review by Ashish-Nanda — APPROVED** [review]
LGTM 🚀

**Pull request merged**